{"_id":"8e37c470-a139-11ec-a08f-53cc30e916bb","postId":"2022_03_cancellation-3-detecting-cancellation-8b525936-6562-34ae-94c1-4dc6f19d6004","postUri":"https://blog.stephencleary.com/2022/03/cancellation-3-detecting-cancellation.html","replyTo":"","authorEmailEncrypted":"","authorEmailMD5":"","authorName":"Patrick Borgogno","authorUri":"","message":"Hey Stephen\r\n\r\nThanks for the great blog and series.\r\nFor the first code block you are explaining how it ignores any OperationCanceledException's.\r\nMight there be a mistake?\r\nThe exception is ignored in the sense of not catching it, it will be thrown further up the chain.\r\nAs I understand it, the \"catch ()\" and \"when ()\" are not in a \"when, then\" but more in a \"if and if\" relationship.\r\n\r\nThanks again\r\n\r\nPatrick","date":"2022-03-11T12:48:31.504Z"}